### PR TITLE
CTCP-4794/4799/4796/4791/4793: Add Another DepartureTransportMeans & Navigation

### PR DESCRIPTION
**Pull request by BledarFeratiEY**

### CODE REVIEW

File: app/models/messages/Consignment.scala [hunk 1]
```diff
@@ -57,7 +57,7 @@ object Consignment {
       (__ \ "modeOfTransportAtTheBorder").readNullable[String] and
       (__ \ "TransportEquipment").readNullable[List[TransportEquipment]] and
       (__ \ "LocationOfGoods").readNullable[LocationOfGoods] and
-      (__ \ "DepartureTransportMeans").readWithDefault[List[DepartureTransportMeans]](Nil).map(_.headOption) and
+      (__ \ "DepartureTransportMeans").readNullable[Seq[DepartureTransportMeans]] and
```
Comment (jackheslop96) [inline app/models/messages/Consignment.scala:60]: I have a feeling the reads and writes can be replaced with the bog standard `implicit val format: Format[Consignment] = Json.format[Consignment]` now

Comment (BledarFeratiEY) [reply app/models/messages/Consignment.scala:60]: Yep this works

**Review by jackheslop96 — DISMISSED** [review]

**Comment by BledarFeratiEY** [comment]
rebuild

**Comment by BledarFeratiEY** [comment]
rebuild

**Review by vajarova — APPROVED** [review]

**Pull request merged**